### PR TITLE
Resolve rejections to lacina promises

### PR DESCRIPTION
**Pull request by thumbnail**
previously only successful promises were resolved, resulting in an infinite block in lacinia. Exceptions are now passed through the promesa promise into the lacinia promise, which is supported by lacinia.

This caused an issue in our setup where exceptions were lost, and workers were blocking on rejected promesa promises.

### CODE REVIEW

File: test/superlifter/lacinia_test.clj [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+(deftest ->lacinia-promise
+  (let [ex (ex-info "Expected" {})]
+    (are [input expected] (= expected
+                             (let [v (promise)]
+                               (on-deliver! (sl/->lacinia-promise input) #(deliver v %))
+                               (deref v 5 ::timeout)))
+      ::value                                           ::value
+      (prom/promise ::value)                            ::value
+      (doto (prom/deferred) (prom/resolve! ::value))    ::value
+      (doto (prom/deferred) (prom/reject! ex))          ex)))
```
Comment (thumbnail) [inline test/superlifter/lacinia_test.clj:17]: This line would fail (with a timeout) if you undo the change in `superlifter.lacinia`

**Comment by oliyh** [comment]
This is great, thank you!
